### PR TITLE
Marking ongoing events

### PR DESCRIPTION
**Pull request by cricketer94**
Still yet to write tests for ongoing events and ongoing function for time util. Will push again

### CODE REVIEW

File: src/main/java/seedu/todo/commons/util/TimeUtil.java [hunk 1]
```diff
@@ -233,6 +233,18 @@ public boolean isOverdue(LocalDateTime endTime) {
...
         return endTime.isBefore(LocalDateTime.now(clock));
     }
+    
+    public boolean isOngoing(LocalDateTime startTime, LocalDateTime endTime){
+        if(endTime == null){
+            logger.log(Level.WARNING, "endTime in isOngoing(...) is null.");
+            return false;
+        }
+        else if(startTime == null){
+            logger.log(Level.WARNING, "startTime in isOngoing(..., ...) is null");
```
Comment (xiendong) [inline src/main/java/seedu/todo/commons/util/TimeUtil.java]: isOngoing(...) is sufficient bah

File: src/main/java/seedu/todo/ui/view/TaskCardView.java [hunk 2]
```diff
@@ -121,12 +121,15 @@ private void displayTags(){
      */
     private void setStyle() {
         boolean isCompleted = task.isCompleted();
-        boolean isOverdue = task.getEndTime().isPresent() && timeUtil.isOverdue(task.getEndTime().get());
+        boolean isOverdue = task.getEndTime().isPresent() && timeUtil.isOverdue(task.getEndTime().get()) && !task.isEvent();
```
Comment (xiendong) [inline src/main/java/seedu/todo/ui/view/TaskCardView.java:124]: isOverdue is true when current time exceeds end time. So I don't think there is a need to check for events?

File: src/main/java/seedu/todo/ui/view/TaskCardView.java [hunk 3]
```diff
@@ -121,12 +121,15 @@ private void displayTags(){
...
+        boolean isOngoing = task.isEvent() && timeUtil.isOngoing(task.getStartTime().get(), task.getEndTime().get());
```
Comment (xiendong) [inline src/main/java/seedu/todo/ui/view/TaskCardView.java]: You should check for .isPresent() before accessing any of the time codes

File: src/main/resources/style/DefaultStyle.css [hunk 4]
```diff
@@ -232,6 +232,34 @@
 }
 
 
+/*Ongoing*/
+.ongoing {
+    -fx-background-color: #388E3C;
+}
+
+
```
Comment (xiendong) [inline src/main/resources/style/DefaultStyle.css]: Delete extra line

File: src/main/resources/style/DefaultStyle.css [hunk 5]
```diff
@@ -232,6 +232,34 @@
...
+.ongoing .dateImage {
+    -fx-image: url("../images/clock_white.png");
+}
+
+.ongoing .locationImage {
+    -fx-image: url("../images/location_white.png");
+}
+
+
```
Comment (xiendong) [inline src/main/resources/style/DefaultStyle.css:261]: Not sure if it is possible to abstract all the styles to a class called "colouredBackground"

File: src/main/java/seedu/todo/ui/view/TaskCardView.java (hunk 2 above)
Comment (ZhangYiJiang) [reply src/main/java/seedu/todo/ui/view/TaskCardView.java:124]: The problem this is trying to solve is that overdue styling was also being applied to events, which is wrong - events can't be "overdue". Model side automatically marks "overdue" events as completed, but the thread is not synchronized to UI side's thread, so for a few seconds incorrect style would be applied to events. This avoids that problem

File: src/main/resources/style/DefaultStyle.css (hunk 5 above)
Comment (ZhangYiJiang) [reply src/main/resources/style/DefaultStyle.css:261]: You guys might want to consider using Looked Up color - see https://docs.oracle.com/javafx/2/api/javafx/scene/doc-files/cssref.html#typecolor (scroll down after the list of named colors)

**Pull request merged**